main.rb

```diff
@@ -0,0 +1,34 @@
+require_relative 'app'
+require_relative 'options'
+
+def run_app(handle_options, _app)
+  menu_options = {
+    '1' => :list_all_books,
+    '2' => :list_all_peoples,
```

Comment:
```suggestion
    '2' => :list_all_people,
```

Before resubmitting make sure your app is working as expected. Thank you!